README.md

```diff
@@ -22,8 +22,8 @@ Usage:
 
 Options: 
         --help                                                   Show help and usage information
-        --cpu.target.load                                        Target level for bursting metrics (int) [default: 60]
-        --cpu.max.load                                           Max level for bursting metrics (int) [default: 80]
+        --burst-target                                           Target level for bursting metrics (int) [default: 60]
```

Comment:
Need to add a couple more parameters here, burst-header (to optionally enable the burst headers) and burst-service (the service name that will be added in the header.

Reply:
Currently burst-headers embedded to these endpoints by default. Expectation is to disable them by default and enable only when explicitly set `--burst-header=true`?

Reply:
yea that's correct. though would prefer if it was `--burst-header` as opposed to `--burst-header=true`. not sure if java has an easy way to support that.

Reply:
added `burst-header` and `burst-service` flags